src/Operations.jl

```diff
@@ -1661,7 +1661,7 @@ function test(ctx::Context, pkgs::Vector{PackageSpec};
             printpkgstyle(ctx.io, :Testing, "Running tests...")
             flush(stdout)
             cmd = gen_test_code(testfile(source_path); coverage=coverage, julia_args=julia_args, test_args=test_args)
-            p = run(ignorestatus(cmd))
+            p = run(pipeline(ignorestatus(cmd), stdout = sandbox_ctx.io))
             if success(p)
                 printpkgstyle(ctx.io, :Testing, pkg.name * " tests passed ")
```

Comment:
After changing this the `printpkgstyle` can't print to `ctx.io`. Does the sub julia process close the io or something?
```
  ArgumentError: ensureroom failed, IOBuffer is not writeable
  Stacktrace:
    [1] ensureroom_slowpath(io::IOBuffer, nshort::UInt64)
      @ Base ./iobuffer.jl:303
    [2] ensureroom
      @ ./iobuffer.jl:325 [inlined]
    [3] unsafe_write(to::IOBuffer, p::Ptr{UInt8}, nb::UInt64)
      @ Base ./iobuffer.jl:424
    [4] write
      @ ./strings/io.jl:244 [inlined]
    [5] print
      @ ./strings/io.jl:246 [inlined]
    [6] with_output_color(f::Function, color::Symbol, io::IOBuffer, args::String; bold::Bool, underline::Bool, blink::Bool, reverse::Bool, hidden::Bool)
      @ Base ./util.jl:79
    [7] #printstyled#884
      @ ./util.jl:129 [inlined]
    [8] printpkgstyle(io::IOBuffer, cmd::Symbol, text::String, ignore_indent::Bool; color::Symbol)
      @ Pkg ~/Documents/GitHub/Pkg.jl/src/utils.jl:5
    [9] printpkgstyle (repeats 2 times)
      @ ~/Documents/GitHub/Pkg.jl/src/utils.jl:3 [inlined]
   [10] (::Pkg.Operations.var"#109#111"{Bool, Cmd, Cmd, Nothing, Pkg.Types.Context, Vector{Tuple{String, Base.Process}}, String, Pkg.Types.PackageSpec})()
      @ Pkg.Operations ~/Documents/GitHub/Pkg.jl/src/Operations.jl:1666
```